core/src/test/resources/test-reference.conf

```diff
@@ -100,67 +100,67 @@
       # this is testing only storage
       storage: "inmem:///proxima_events"
       type: primary
-      access: "commit-log"
+      access: commit-log
```

Comment:
MINOR: Is this " removal needed? Can it be more backward compatible?

Reply:
It is backwards compatible. That is feature of hocon, these two statements are actually equal, so this is just a cleanup.